src/iosrv/init.c

```diff
@@ -756,28 +753,13 @@ parse(int argc, char **argv)
 			printf("\"-c\" option is deprecated, please use \"-t\" "
 			       "instead.\n");
 		case 't':
-			nr = strtoul(optarg, &end, 10);
-			if ((end == optarg) || (nr == ULONG_MAX)) {
-				rc = -DER_INVAL;
-				break;
-			}
-			nr_threads = nr;
+			nr_threads = atoi(optarg);
```

Comment:
I don't have strong opinion on this, just a point that -
       The atoi() function converts the initial portion of the string
       pointed to by nptr to int.  The behavior is the same as
           strtol(nptr, NULL, 10);
       except that atoi() does not detect errors.

Reply:
I was trying to strike a balance between addressing the actual problem that Coverity flagged (fix the fact that we were using the wrong data type to hold the result, mixing long and unsigned long) and minimizing code changes.  I was thinking that it could be appealing to unify the approach to getting these argument values, so I went with the simple atoi(optarg) option.  If you think we'd be better off using a different approach, I am open to that.

Reply:
is it type mismatch between nr_threads and return value of strtoul()? can change type of nr_threads, or not sure can it silent Coverify warning by adding force type convert?
original strtoul() maybe a little better as it can detect invalid digit? but looks not a big deal.